.github/actions/markdown-lint/action.yml

```
@@ -0,0 +1,35 @@
+name: Markdown Linting
+description: >
+  Lints markdown files by wrapping the markdownlint-cli2 action
+  (https://github.com/DavidAnson/markdownlint-cli2-action).
+
+# When updating defaults here also update them in the `markdown-lint` workflow.
+inputs:
+  config:
+    description: Path to a file to use for the base configuration object (defaults to "lombiq.markdownlint.json").
+    default: 'lombiq.markdownlint.json'
+    required: false
+  fix:
+    description: Whether to fix issues automatically for the rules that support it (any truthy value enables).
+    default: ''
+    required: false
+  globs:
+    description: Glob expression(s) of files to lint (newline-delimited).
+    default: '**/*.{md,markdown}'
+    required: false
+  separator:
+    description: String to use as a separator for the "globs" input (defaults to newline).
+    default: "\n"
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Markdown Linting
+    # When upgrading, also update Lombiq.NodeJs.Extensions to use the corresponding version of the markdownlint package.
+      uses: DavidAnson/markdownlint-cli2-action@b4c9feab76d8025d1e83c653fa3990936df0e6c8 # v16.0.0
```

imo the comment should have the same indentation as the `uses`.
```suggestion
    - name: Markdown Linting
      # When upgrading, also update Lombiq.NodeJs.Extensions to use the corresponding version of the markdownlint package.
      uses: DavidAnson/markdownlint-cli2-action@b4c9feab76d8025d1e83c653fa3990936df0e6c8 # v16.0.0
```

---

Yep, thanks!